Sources/protoc-gen-swiftgrpc/Generator.swift

```diff
@@ -106,35 +99,14 @@ class Generator {
     println()
     
     if options.generateClient {
-      if options.generateImplementations {
-        for service in file.services {
-          self.service = service
-          printClient(asynchronousCode: options.generateAsynchronous,
-                      synchronousCode: options.generateSynchronous)
-        }
-      }
-
-      if options.generateTestStubs {
-        if options.generateNIOImplementation {
-          fatalError("Generating test stubs is not yet supported for SwiftGRPC-NIO.")
-        }
-        for service in file.services {
-          self.service = service
-          printCGRPCClientTestStubs(asynchronousCode: options.generateAsynchronous,
-                                    synchronousCode: options.generateSynchronous)
-        }
+      for service in file.services {
+        self.service = service
+        printClient()
       }
     }
     println()
     
     if options.generateServer {
-      if options.generateTestStubs && options.generateNIOImplementation {
```

Comment:
Any idea why this was in here? I did think we have server test stubs already. Or do we have them, but don't need to generate any code for them?

Reply:
No idea, git tells me you wrote this line though, so you may be better positioned to answer that question! 🙃

As far as I can tell there are no test stubs for NIO.

Reply:
Right, I think the test stub so far simply has you use `UnaryResponseCallContextTestStub` and the like directly; no need to generate code for these.